commands/channel.js

```diff
@@ -613,7 +616,63 @@ module.exports = {
 						await interaction.message.edit({ components: buttons });
 					});
 			}
+			case 'set_jtc_status': {
+				const modal = new ModalBuilder()
+					.setCustomId(global.getButtonIdString('channel', 'set_jtc_status', [interaction.channel.id]))
+					.setTitle('Voice Channel Status');
+				const statusInput = new TextInputBuilder()
+					.setCustomId('status')
+					.setLabel('Enter the new voice channel status:')
+					.setMaxLength(30)
+					.setRequired(false)
+					.setStyle(TextInputStyle.Short);
+				const actionRow = new ActionRowBuilder()
+					.addComponents(statusInput);
+				modal.addComponents(actionRow)
+
+				return interaction.showModal(modal);
+			}
+			default:
+				return global.ephemeralReply(interaction, 'Invalid request.');
+		}
+		return Promise.reject();
+	},
+	async modal(interaction, guild, member, perm, subCommand, args) {
+		if (args.length < 1) {
+			return global.ephemeralReply(interaction, 'Invalid request.');
+		}
+		switch (subCommand) {
+			case 'set_jtc_status': {
+				const createdBy = global.tempChannelCreatedBy(interaction.channel.id);
+				if (!createdBy) {
+					return global.ephemeralReply(interaction, 'This is not a JTC channel');
```

Comment:
What does the interaction reply look like for a modal?

Reply:
It’s a message reply in the channel like the other buttons or command.